snippets/Timer.md

```diff
@@ -0,0 +1,49 @@
+### Timer
+
+Renders a timer component.
+
+- The timer state is initially set to zero 
+- When the `Tick!` button is clicked, `timer` is incremented periodically at the given `interval`
+- When the `Reset` button is clicked, the value of the timer is set to zero and the `setInterval` is cleared
+- The `setInterval` is cleared once the desired `time` is reached
+- `time` and `interval` are the required props
+
+```jsx
+class App extends Component {
```

Comment:
The name of the component does not match the snippet name.

Reply:
Fixed